### ISSUE TITLE
Foreign key constraints are treated as unique keys

### ISSUE THREAD

**Issue by togana** [k1LoW/tbls#239]
### What happened

If you add a unique attribute and a foreign key constraint at the same time, the foreign key constraint information will be lost.

### What you expected to happed

If you check [category_info.md](https://github.com/togana/example-tbls/blob/master/dust/dbdoc/category_info.md), sql is correct below.

```sql
CREATE TABLE `category_info` (
  `id` int(11) NOT NULL,
  `category_id` int(11) NOT NULL,
  `group_name` varchar(255) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `category_info_category_id_unique` (`category_id`),
  CONSTRAINT `category_info_category_id_foreign` FOREIGN KEY (`category_id`) REFERENCES `categories` (`id`) ON DELETE CASCADE
) ENGINE=InnoDB DEFAULT CHARSET=utf8
```

If you check the constraints, foreign will be treated as UNIQUE.

| Name | Type | Definition |
| ---- | ---- | ---------- |
| category_info_category_id_foreign | UNIQUE | UNIQUE KEY category_info_category_id_foreign (category_id) |
| category_info_category_id_unique | UNIQUE | UNIQUE KEY category_info_category_id_unique (category_id) |
| PRIMARY | PRIMARY KEY | PRIMARY KEY (id) |

I want it to be recognized as a foreign key.

```diff
| Name | Type | Definition |
| ---- | ---- | ---------- |
- | category_info_category_id_foreign | UNIQUE | UNIQUE KEY category_info_category_id_foreign (category_id) |
+ | category_info_category_id_foreign | FOREIGN KEY | FOREIGN KEY (category_id) REFERENCES categories (id)
| category_info_category_id_unique | UNIQUE | UNIQUE KEY category_info_category_id_unique (category_id) |
| PRIMARY | PRIMARY KEY | PRIMARY KEY (id) |
```

### What stacktrace or error message from tbls did you see?

No error message.

### Anything else we need to know?

Reproduce with the data defined in the repository below.

https://github.com/togana/example-tbls

Execute the following to generate a document.

```sh
$ docker-compose up -d mysql
$ sleep 5
$ docker-compose run --rm tbls
```

### Environment

- `tbls version`

use docker image tag latest.

```
$ docker image ls k1low/tbls
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
k1low/tbls          latest              303cdf6d0395        2 months ago        39.4MB
```

- Database version 

```
$ docker image ls mysql:5.7
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
mysql               5.7                 cd3ed0dfff7e        7 months ago        437MB
```

- OS

macOS Calalina

- Shell

zsh

- Stacktrace or error message from tbls ( If you can, set `DEBUG=1`)
- Config ( .tbls.yml )

**Comment by togana** [comment]
I'm sorry.
It was fixed when I checked it with the version of 1.38.3.

**Comment by k1LoW** [comment]
@togana Thank you for your report !!

> It was fixed when I checked it with the version of 1.38.3.

👍 👍 👍 👍 👍 👍
